### ISSUE TITLE
Unable to upload torrent file

### ISSUE THREAD

**Issue by c1tyguide** [killemov/Shift#31]
Trying upload torrent file nothing happens
In console gets:
```
prototype.js:1737 
                
       POST http://192.168.xxx.xxx:9091/transmission/rpc 409 (Conflict)
request @ prototype.js:1737
initialize @ prototype.js:1702
(anonymous) @ prototype.js:447
klass @ prototype.js:80
doRequest @ shift.js:955
(anonymous) @ shift.js:6398
(anonymous) @ prototype.js:7299
fireEvent_DOM @ prototype.js:7158
fire @ prototype.js:7147
_methodized @ prototype.js:456
fireContentLoadedEvent @ prototype.js:7334
shift.js:776 Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at persistPath (shift.js:776:28)
    at HTMLButtonElement.<anonymous> (shift.js:5433:72)
    at HTMLButtonElement.<anonymous> (prototype.js:7283:15)
```

**Comment by killemov** [comment]
The 409 error is standard. See https://github.com/killemov/Shift/issues/30#issuecomment-2428835369

But it may have triggered an unwanted side-effect.

Do you get this error all the time when uploading? I just get the 409 very once in a while.

**Comment by c1tyguide** [comment]
Yes, it permanent. I using Version: 1.2 / 2.84 (14307) on NAS WD MyBook Live with Debian Jessie under chroot.

**Comment by killemov** [comment]
@c1tyguide I could just add a check to verify `settings.paths` exists but that would be pointless if the entire settings object is empty. Can you add a breakpoint on line 776 and see if the settings object has any data at all at that moment? If that is the case can you navigate to [Session] -> [Shift], push [Scrape Paths] and then push [Apply]? And then test again?

**Comment by c1tyguide** [comment]
> @c1tyguide I could just add a check to verify `settings.paths` exists but that would be pointless if the entire settings object is empty. Can you add a breakpoint on line 776 and see if the settings object has any data at all at that moment? If that is the case can you navigate to [Session] -> [Shift], push [Scrape Paths] and then push [Apply]? And then test again?
> 
> This issue could be related to #30.

Thanks a lot! With latest Shift adding torrents works like a charm! So not necessary to debug code?

**Comment by killemov** [comment]
> Thanks a lot! With latest Shift adding torrents works like a charm! So not necessary to debug code?

If you like it put a [★](https://github.com/killemov/Shift/stargazers) on it. For this particular issue no further investigation is necessary. Feedback and ideas are always welcome.

So also fixed with #30.
